config.py

```diff
@@ -22,6 +31,24 @@
 
 # Node provier configuration
 # You can also get a trader node https://docs.chainstack.com/docs/solana-trader-nodes
-MAX_RPS = 25  # TODO: not implemented. Max RPS to avoid rate limit errors
+MAX_RPS: int = 25  # TODO: not implemented. Max RPS to avoid rate limit errors
 PUBLIC_RPC_ENDPOINT = "https://api.mainnet-beta.solana.com"
```

Comment:
Let's not unnecessarily overload the public infra here so that people don't start their FAFO without the basic understanding of this one and hurt the public good.
So no default RPCs please. And let's have a comment to the tune of "Tested with Chainstack nodes + link to concole.chainstack.com; you can of course any node provider here"